test/parallel/test-outgoing-message-pipe.js

```diff
@@ -8,8 +8,9 @@ const OutgoingMessage = require('_http_outgoing').OutgoingMessage;
 const outgoingMessage = new OutgoingMessage();
 assert.throws(
   common.mustCall(() => { outgoingMessage.pipe(outgoingMessage); }),
-  (err) => {
-    return ((err instanceof Error) && /Cannot pipe, not readable/.test(err));
-  },
+  common.expectsError({
```

Comment:
`common.expectsError` can take a function and wraps it in `assert.throws`, can you simplify this to

```js
common.expectsError(
  () => { outgoingMessage.pipe(outgoingMessage); },
  {
    code: 'ERR_STREAM_CANNOT_PIPE',
    type: Error
  }
);
```

?

Reply:
Done!